packages/app/src/server/models/user.js

```diff
@@ -689,6 +690,13 @@ module.exports = function(crowi) {
     user.save();
   };
 
+  userSchema.statics.findUserBySlackId = function(slackId) {
+    if (slackId == null) {
+      return Promise.resolve(null);
+    }
+    return this.findOne({ slackId });
+  };
+
```

Comment:
Please use `async` function instead.

Reply:
Additionally, I think there is no use case that we call `findUserBySlackId` for now due to the query optimization issue(we can reduce the number of query by fetching all corresponding growi users at once). So, please implement `findUsersBySlackIds` method that takes an array of slack ID as a parameter and returns an array of growi users.